.travis.yml

```diff
@@ -86,8 +86,7 @@ install:
 
   - |
     if [[ $BUILD_DOCS == true ]]; then
-      pip install $PRE numpydoc ipython jsonschema mistune colorspacious
-      pip install -q $PRE linkchecker
+      pip install $PRE numpydoc ipython colorspacious
```

Comment:
I have removed jsonshema and mistune they are not needed anymore. They were only there because ipython had a bud in its dependencies